NOTES.md

```diff
@@ -1,3 +1,191 @@
+# Release 1.16.0 (Stand By Me)
+
+Released on 2015-01-28
+
+The highlights of this release are numerous and exciting. Read the
+[release blog post][1.16-blog] for more details.
+
+[1.16-blog]: http://rethinkdb.com/blog/1.16-release/
+
+There may or may not be API-breaking changes in this release.
+
+## New Features ##
+
+* ReQL admin: a new cluster management and monitoring API (#2169)
+  * Added a special system database named `rethinkdb`
+    * The `table_config` table allows changing a table's configuration (#2870)
+    * The `server_config` table allows listing and managing servers in the cluster (#2873)
+    * The `db_config` table allows listing and renaming databases (#151, #2871)
+    * The `cluster_config` table contains cluster-wide settings (#2924)
+    * The `table_status` table displays each table's availability and replication status (#2983, #3269)
+    * The `server_status` table contains information about the servers in the cluster (#2923, #3270)
+    * The `current_issues` table lists issues affecting the cluster and suggests solutions (#2864, #3258)
+    * The `jobs` table lists running queries and background tasks (#3115)
+    * The `stats` table contains real-time statistics about the server (#2885)
+    * The `logs` table provides access to the server logs (#2884)
+    * The hidden `_debug_table_status` table contains internal debugging information (#2901)
+    * The hidden `_debug_stats` table contains internal statistics (#3385)
+    * The `identifier_format` optional argument to `table` switches between table names (when set to "name") and table identifiers (when set to "uuid") (#3266)
```

Comment:
This changes the identifier format for servers and databases in addition to tables. Also, I'd say "table UUIDs" instead of "table identifiers", since we're using "identifier" as a generic term that can mean a name or a UUID.